MUICore/justfile

```diff
@@ -2,7 +2,7 @@ generate:
   python3 setup.py generate
 
 build: generate
-  python3 setup.py build
+  python3 setup.py build_binary
 
 install: build
-  python3 setup.py install
+  cp dist/muicore_server ../MUI/os/linux_x86_64/
```

Comment:
i'd imagine eventually when MUICore has releases, the dist/ directory will be added to .gitignore and users would just download the server binary and the proto spec file directly

Reply:
I don't think we need to version control the `shiv` binary in `MUICore/dist/muicore_server`, do we? We really only care about the one in the Ghidra subdirectory

Reply:
yeah not for now, though if MUICore becomes a standalone repo there might be different plugins at different stages of development using different versions of the binary

Reply:
Yeah, each plugin will need to specify a version range for MUICore that they support. MUICore should use semantic versioning (when things are more stable) to let plugins know when MUICore makes backwards-incompatible changes

Reply:
yep that's in line with my thinking. I'm not too experienced with this - does generating the binary into a dist folder suggest otherwise?

Reply:
oh lol on mobile so just saw the comment below derp, got it!